### PR TITLE
Add Graphviz

### PR DESCRIPTION
**Pull request by HackerFoo**
I've created patches to compile Graphviz for Termux.

### CODE REVIEW

File: packages/graphviz/fix-edgepaintmain-include.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- graphviz-2.38.0/cmd/edgepaint/edgepaintmain.c	2014-04-13 13:40:25.000000000 -0700
++++ src/cmd/edgepaint/edgepaintmain.c	2016-04-03 10:46:38.532882314 -0700
+@@ -27,7 +27,7 @@
+ #include <cgraph.h>
+ #include <agxbuf.h>
+ #include <ingraphs.h>
+-#include <pointset.h>
++#include "pointset.h"
```
Comment (fornwall) [inline packages/graphviz/fix-edgepaintmain-include.patch]: Is this change really needed?

**Comment by fornwall** [comment]
Nice!

See my comments above.

Do you want to stand as the package maintainer, so you can be poked if there are issues found in the package? Add `TERMUX_PKG_MAINTAINER="Dustin DeWeese @HackerFoo"` if so!

File: packages/graphviz/fix-edgepaintmain-include.patch (hunk 1 above)
Comment (HackerFoo) [inline packages/graphviz/fix-edgepaintmain-include.patch]: All changes were necessary at the time I wrote these patches. After madly hacking away to get Graphviz to compile, I removed any changes that wouldn't break the build.

File: packages/graphviz/general-free.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- src/lib/sparse/general.h.orig	2016-04-01 08:13:25.222810595 -0700
++++ src/lib/sparse/general.h	2016-04-01 08:14:12.770312988 -0700
+@@ -14,6 +14,8 @@
+ #ifndef GENERAL_H
+ #define GENERAL_H
+ 
++void free(void *ptr);
```
Comment (HackerFoo) [inline packages/graphviz/general-free.patch]: Honestly, I can't exactly remember why I needed to declare `free()`, but the compiler was complaining that it wasn't declared and it was easier to add than to figure out why.

**Comment by fornwall** [comment]
@HackerFoo Great work, merged and available from the apt repo with:

    apt update && apt install graphviz
